### PR TITLE
Updated to new API version and Weather API Configures

### PR DESCRIPTION
**Pull request by bifrost-trojan**

### CODE REVIEW

**Review by devashishkumar709 — APPROVED** [review]

File: ReadMe.md [hunk 1]
```diff
@@ -1,15 +1,70 @@
 <p align="center">
   <img width="500" src="https://user-images.githubusercontent.com/54733680/152427341-ae5d1b11-aa7e-448d-ac8a-9daef1f5258c.jpg">
 </p>
-<h2 align=center>This Application is Developed Using JavaScript for HarmonyOS.</h2>
+<h2 align=center>This Application is Developed Using JavaScript for HarmonyOS .</h2>
```
Comment (archanaatr) [inline ReadMe.md]: Please dont use HarmonyOS string Readme, instead mention as OpenHarmony.... Readme files please discuss and take from Vishnu

File: entry/src/main/js/MainAbility/app.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export default {
```
Comment (archanaatr) [inline entry/src/main/js/MainAbility/app.js:16]: Pls add copyright details to all new js / css files

**Review by archanaatr — COMMENTED** [review]
Please check the comments.

File: ReadMe.md (hunk 1 above)
Comment (sahilchutani) [reply ReadMe.md]: Ok Mam, we have changed that.

File: entry/src/main/js/MainAbility/app.js (hunk 2 above)
Comment (sahilchutani) [reply entry/src/main/js/MainAbility/app.js:16]: Added copyright details to all files.

**Comment by sahilchutani** [comment]
> Please check the comments.

Done

File: entry/src/main/js/MainAbility/pages/index/index.js [hunk 3]
```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright (C) 2022 Sahil Chutani and Devashish
```
Comment (archanaatr) [inline entry/src/main/js/MainAbility/pages/index/index.js]: 1. Please include multi line copyright - start line /* and */ in the last line
2. copyright not required in hml files, u can remove
3. copyright missing in few files like app.js pls recheck once
4. You need to add app lib groups copyright, right now it is like this, Copyright (C) 2022 Sahil Chutani and Devashish    pls update

Comment (bifrost-trojan) [reply entry/src/main/js/MainAbility/pages/index/index.js]: Done

**Comment by devashishkumar709** [comment]
Ma'am please check we have updated.

**Review by archanaatr — APPROVED** [review]
ok

**Pull request merged**